docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,128 @@
+---
+comments: true
+---
+
+# Adding a New Metrics Provider for Placeholder Endpoint
+
+In this guide, we will create a placeholder provider.
+The following steps are a starting point to create your own custom provider:
+
+1. Fork and clone the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
+  for more information [checkout this link](https://keptn.sh/stable/docs/contribute/general/git/fork-clone/)
+
+2. **Define the Provider Type:** In the `metrics-operator/controllers/common/providers/common.go` file,
+ define the constant `KeptnPlaceholderProviderType`.
+  In our example we use `"placeholder"`.
+
+    ```go
+    const KeptnPlaceholderProviderType = "placeholder"
+    ```
+
+3. **Implement the Provider:** Create your own new folder inside the
+[metrics-operator/controllers/common/providers](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/controllers/common/providers),
+ matching the new provider name: in our case placeholder and a new Go package for the placeholder provider.
+  This package should contain
+ a struct that implements the `KeptnSLIProvider` interface.
+  To fully implement the `KeptnSLIProvider` interface, it's necessary to implement the following functions.
+  `EvaluateQuery`(Fetches metric values from the provider)
```

Comment:
The list of functions needs to be an unordered list beginning here.  And the coding for that is a little tricky with our new tools -- writing that up is the next thing on my list.  Basically, you need to indent 3 spaces in from the text that precedes it and turn off markdownlint for the list.  Let me quickly write up that piece of my doc and push it so you can follow it.  And then I'll come back and review the rest of this.

Very nice job on the documentation, BTW.  It has a good, logical flow.  Kudos!

Reply:
Thanks 🙂